From drwyrm@gmail.com  Thu Jul  3 15:36:03 2008
From: Daniel Ribeiro <drwyrm@gmail.com>
Date: Tue, 01 Jul 2008 23:09:27 -0300
Subject: USB: remove board-specific UP2OCR configuration from pxa27x-udc
To: Greg KH <greg@kroah.com>
Cc: linux-usb@vger.kernel.org, David Brownell <david-b@pacbell.net>,  Robert Jarzmik <robert.jarzmik@free.fr>
Message-ID: <486AE357.5030804@gmail.com>


Remove the board-specific UP2OCR configuration from the
pxa27x-udc driver.

Signed-off-by: Daniel Ribeiro <drwyrm@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/pxa27x_udc.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/gadget/pxa27x_udc.c
+++ b/drivers/usb/gadget/pxa27x_udc.c
@@ -1575,7 +1575,6 @@ static void udc_enable(struct pxa_udc *u
 {
 	udc_writel(udc, UDCICR0, 0);
 	udc_writel(udc, UDCICR1, 0);
-	udc_writel(udc, UP2OCR, UP2OCR_HXOE);
 	udc_clear_mask_UDCCR(udc, UDCCR_UDE);
 
 	clk_enable(udc->clk);
